.travis.yml

```diff
@@ -2,11 +2,15 @@ language: python
 python:
   - "3.6"
 cache: pip
+services:
+  - mysql
+before_install:
+  - mysql -e 'CREATE DATABASE IF NOT EXISTS pytest_testdb;'

```

Comment:
This is an essential step to use MySQL in Travis CI as described in the [documentation](https://docs.travis-ci.com/user/database-setup/#mysql).